certificates/jwt/super-admin-metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "sub": "kuksa.val",
+  "iss": "Eclipse KUKSA Dev",
+  "admin": true,
+  "metadata": "rw",
```

Comment:
modifyTree? modifyVSSTree?

Reply:
really necessary to change?

Reply:
I think `modifytree`  describes more clearly what we are doing here. But now I am puzzled about "r" .. are we also protecting getMetadata now?

Reply:
no, default always read-only. But it is also possible to extend it

Reply:
I update the key work to `modifyTree`. Question: do we need two token?